src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$primary-color: #fff;
```

Comment:
the variables name for colors following good practice should start by color word e.g.:

$color-primary
$color-input-background
$color-address-primary

itp.